providers/src/airflow/providers/google/cloud/operators/alloy_db.py

```diff
@@ -330,13 +331,18 @@ def __init__(
         self.update_mask = update_mask
         self.allow_missing = allow_missing
 
-    def execute(self, context: Context) -> Any:
-        message = (
-            "Validating an Update AlloyDB cluster request."
-            if self.validate_request
-            else "Updating an AlloyDB cluster."
+    def execute(self, context: Context) -> proto.Message | None:
```

Comment:
Probably should be `dict` instead of Message

Reply:
Fixed